cmd/generator/main.go

```diff
@@ -0,0 +1,78 @@
+package main
+
+import (
+	"os"
+)
+
+func main() {
+	argsWithProg := os.Args
+
+	if len(argsWithProg) != 2 {
+		panic("Wrong amount of arguments!")
+	}
+
+	if len(argsWithProg[1]) == 0 {
+		panic("Given regex is empty!")
+	}
+
+	regex := argsWithProg[1]
+
+	_, err := os.Stdout.WriteString((generateCode(regex) + "\n"))
+	if err != nil {
+		panic("Error writing to stdout")
+	}
+}
+
+func generateCode(regex string) string {
+	afterLastBrace, beforeCurrentBrace, indentations, result := regex, "", "", "rex.New(\n"
+
+	currentOpenBraceIndex, bracesCounter := 0, 0
+
+	for index, runeValue := range regex {
```

Comment:
It is usual to use `i` for index. Long names are good, but there is [a general approach](https://github.com/golang/go/wiki/CodeReviewComments#variable-names).

Reply:
Ok :)